app/models/manageiq/providers/ansible_tower/configuration_manager/configuration_script/api_create.rb

```diff
@@ -0,0 +1,36 @@
+class ManageIQ::Providers::AnsibleTower::ConfigurationManager::ConfigurationScript
+  module ApiCreate
+    def create_in_provider(manager_id, params)
+      manager = ExtManagementSystem.find(manager_id)
+      job_template = manager.with_provider_connection do |connection|
+        connection.api.job_templates.create(params)
+      end
+
+      # Get the record in our database
+      # TODO: This needs to be targeted refresh so it doesn't take too long
```

Comment:
Then, you should just pass in the target instead of the manager.  Without targeted refresh, it will resolve to the manager.  When targeted refresh is available, it will take the target into account.